src/Parser/MarksmanshipHunter/CombatLogParser.js

```diff
@@ -24,12 +27,14 @@ class CombatLogParser extends CoreCombatLogParser {
     alwaysBeCasting: AlwaysBeCasting,
     castEfficiency: CastEfficiency,
     cooldownTracker: CooldownTracker,
+    vulnerableTracker: VulnerableTracker,
     vulnerabluptime: VulnerableUpTime,
```

Comment:
I must've missed that earlier, should be `vulnerableUptime`.

Reply:
It's a module made by JLassie, but I am considering completely scratching it, it is currently a somewhat useless metric -- just feels rude, but I'll update that name for now.

Reply:
Usefulness of the module is not up to me to judge, I just wanted to point out the name :) So `vulnerableUptime: VulnerableUpTime,` just to be clear which one I meant :)